examples/kmm-sample/settings.gradle.kts

```diff
@@ -12,6 +12,7 @@ pluginManagement {
         gradlePluginPortal()
         google()
         mavenCentral()
+        mavenLocal()
```

Comment:
Needed to support building examples locally.

Reply:
If you wan't to build example locally you can just re-insert the 
```
includeBuild("../../packages")
```
In line 3!? Or are there other uses cases for doing local builds?